src/poker_now_log_converter/game.py

```diff
@@ -124,7 +187,7 @@ def __init__(self, poker_now_log: List[List[str]], currency: str = "USD", timezo
                 if p_seat:
                     p_seat.seat_did_bet = True
 
-            elif "bets" in line:
+            elif "\" bets" in line:
```

Comment:
Thanks for this fix! 

When I looked at this issue I found that double quotes are an allowed username characteristic, so we could always end up with someone calling themselves `"some guy" folds now` or something and still run into this behaviour. I had started writing a bigger tokenizing feature but never finished it :(

I like this fix for now though, it cuts out 99.9% of cases so that's a nice win.